app/views/assessor_interface/review_verifications/index.html.erb

```diff
@@ -7,17 +7,22 @@
   The following verification tasks have been flagged for review:
 </p>
 
-<%= govuk_table do |table|
+<%= render(TaskList::Component.new(
   if @professional_standing_request.present?
-    table.with_caption(text: "LoPS")
-    table.with_body do |body|
-      body.with_row do |row|
-        row.with_cell { govuk_link_to region_certificate_name(@application_form.region), [:review, :assessor_interface, @application_form, @assessment, :professional_standing_request] }
-        row.with_cell { render(StatusTag::Component.new(@professional_standing_request.review_status)) }
-      end
-    end
+    [
+      {
+        title: "LoPS",
+        items: [
+          {
+            name: (region_teaching_authority_name(@application_form.region) == "teaching authority") ? "Contact relevant competent authority" : region_teaching_authority_name(@application_form.region),
```

Comment:
I think it's better to have the default in the helper itself as a second argument, and then pass that in, I think you had that at one point in this PR.